packages/react-hooks/src/useScrollDirection.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * Copyright IBM Corp. 2018, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { useState } from 'react';
+import { usePrevious } from './usePrevious';
+import { useWindowScroll } from './useWindowScroll';
```

Comment:
it looks like this file is missing?

```suggestion
import { useWindowScroll } from './useWindow';
```

Reply:
Whoops, should be `useWindow` like you noted lol